ambari-infra/ambari-infra-manager/pom.xml

```diff
@@ -17,14 +17,14 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <artifactId>ambari-infra</artifactId>
     <groupId>org.apache.ambari</groupId>
     <version>2.7.3.0.0</version>
   </parent>
   <name>Ambari Infra Manager</name>
-  <url>http://maven.apache.org</url>
+  <url>https://maven.apache.org</url>
```

Comment:
looks like this URL does not make any sense. Probably we should remove it


Element | Description
-- | --
packaging | defines packaging type such as jar, war etc.
name | defines name of the maven project.
url | defines url of the project.
dependencies | defines dependencies for this project.
dependency | defines a dependency. It is used inside dependencies.
scope | defines scope for this maven project. It can be compile, provided, runtime, test and system.